gh.go

```diff
@@ -34,6 +34,16 @@ func Exec(args ...string) (stdOut, stdErr bytes.Buffer, err error) {
 	return run(path, nil, args...)
 }
 
+// Exec gh command interactively with provided arguments.
+func ExecInteractive(args ...string) (err error) {
+	path, err := path()
+	if err != nil {
+		err = fmt.Errorf("could not find gh executable in PATH. error: %w", err)
```

Comment:
I understand that this basically copies the error message from `Exec()`, but we could maybe improve this error message in both functions.

The original error from `exec.LookPath` (which `safeexec` uses under the hood) will be something like:
```
exec: "gh": executable file not found in $PATH
```
After being wrapped, the full message will be:
```
could not find gh executable in PATH. error: exec: "gh": executable file not found in $PATH
```
I don't think the extra explanation added by wrapping adds any value here. I'd vote to avoid wrapping this error message altogether (and in the `Exec()` function too).

Reply:
Done